modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
also storing data has a cost, and we're talking about hundreds of millions of instances here.